frontend/src/constants/actionTypes.tsx

```diff
@@ -27,11 +27,13 @@ export const ADD_REQUEST_ACCESS = 'addRequestAccess';
 export const GET_REQUEST_ACCESS = 'getRequestAccess';
 export const UPDATE_REQUEST_ACCESS_ADMIN = 'updateRequestAccessAdmin';
 export const UPDATE_REQUEST_ACCESS_STATUS_ADMIN = 'updateRequestAccessStatusAdmin';
```

Comment:
The more I see this file the more I cringe.  I think we need to quickly start using typescript enums.

https://medium.com/@Alserda/readable-ts-enum-actiontypes-in-react-redux-c105fa0a7c9b

Reply:
I have other ideas to clean this up. I introduced a new hook call useApi, and we can extend it to have feature-focused actions. I will present my idea to the team on Friday if we have enough time